### PR TITLE
Update policies-vulnerability-patch.md

### PR DESCRIPTION
**Pull request by tcalotta**
Added additional CVEs to the known list

### CODE REVIEW

File: docs/vendor/policies-vulnerability-patch.md [hunk 1]
```diff
@@ -64,4 +65,6 @@ CVE-2014-9357 |	A flaw was found in the way the Docker service unpacked images o
...
+|CVE-2020-27847| A vulnerability exists in the SAML connector of the github.com/dexidp/dex library used to process SAML Signature Validation. This flaw allows an attacker to bypass SAML authentication. The highest threats from this vulnerability are to confidentiality, integrity, and system availability. This flaw affects dex versions before 2.27.0. | This is a false positive from Trivy. This vulnerability applies to Dex version 2.27.0 and earlier, whereas Replicated is using Dex version 2.32.0. Dex does not follow Go's semantic versioning convention, which may explain the erroneous result from Trivy. | https://github.com/replicatedhq/kots/pull/2934 https://github.com/aquasecurity/trivy/issues/2433 https://github.com/dexidp/dex/issues/1578#issuecomment |
```
Comment (paigecalvert) [inline docs/vendor/policies-vulnerability-patch.md:68]: >This flaw affects dex versions before 2.27.0.

Should this say "dex version 2.27.0 and earlier"?

I see that a couple lines down it says "This vulnerability applies to Dex version 2.27.0 and earlier, ...", so just want to clarify that it does in fact include 2.27.0.

File: docs/vendor/policies-vulnerability-patch.md [hunk 2]
```diff
@@ -64,4 +65,6 @@ CVE-2014-9357 |	A flaw was found in the way the Docker service unpacked images o
...
+|CVE-2021-38561 | Due to improper index calculation, an incorrectly formatted language tag can cause Parse to panic, due to an out of bounds read. If Parse is used to process untrusted user inputs, this may be used as a vector for a denial of service attack. | KOTS does not accept generic user input, only Kubernetes manifests which are then passed to Kustomize. Additionally, there is never a direct interface to golang’s text library | https://www.mend.io/vulnerability-database/CVE-2021-38561 |
```
Comment (paigecalvert) [inline docs/vendor/policies-vulnerability-patch.md:69]: >can cause Parse to panic

Could you replace "panic" with more precise language about what happens? Does Parse throw a particular error/fail in some way?

**Review by paigecalvert — COMMENTED** [review]
Thank you @tcalotta! Just two comments

File: docs/vendor/policies-vulnerability-patch.md (hunk 1 above)
Comment (tcalotta) [reply docs/vendor/policies-vulnerability-patch.md:68]: I don't actually know if 2.27.0 is included. The summary is copied directly from the CVE, ambiguous phrasing and all, so it's possible I misinterpreted. At any rate, it's moot as we're running 2.32.0

File: docs/vendor/policies-vulnerability-patch.md (hunk 2 above)
Comment (tcalotta) [reply docs/vendor/policies-vulnerability-patch.md:69]: The summary is verbatim from the original CVE so I unfortunately don't have any further info. Seems this wording was posted by the original golang dev who discovered this and has been carried through all the way to the CVE.

**Review by paigecalvert — APPROVED** [review]

**Comment by paigecalvert** [comment]
Thanks for clarifying @tcalotta --I misunderstood where these descriptions were coming from.

**Comment by tcalotta** [comment]
No worries. I'll clarify:
Cve ID and the summary are typically taken verbatim from the cve. Sometimes I'll edit the summary for brevity. The rationale is entirely my work. Further reading are helpful/relevant links.
________________________________

**Review by marccampbell — APPROVED** [review]

**Pull request merged**